src/content/en/2020/accessibility.md

```diff
@@ -8,17 +8,437 @@ authors: [oluoluoxenfree, alextait1]
+  height=TODO
+) }}
+
+For desktop pages, 7.00% have at least one element with a `role=”tablist”` whereas there only 5.79% of pages have elements with a `role=”tab”` and 5.46% of pages have elements with a `role=”tabpanel”`. This suggests that the pattern may only be partially implemented. Even if there is dynamic rendering at play for some of the tab/tabpanel elements, the currently visible or first tab/tabpanel would theoretically be in the DOM on page load.
+
+##### Presentation
+
+When an element has been given a `role=”presentation”` its semantics are stripped away, for both the element it is assigned to and its required children. For example, tables and lists both have required children, so if the parent has a `role=”presentation”` this essentially cascades to the child elements, which will also have their semantics stripped. Removing an element’s semantics means that it is no longer that element in any capacity except for its visual appearance. For example, a list with a `role=”presentation”` will no longer communicate any information to a screen reader about the list structure.
+
+A common usage of this attribute is for `<table>` elements that have been used for layout rather than  for tabular data. We do not recommend using tables in this way. For layout, we have powerful CSS tools today such as flexbox and CSS grid. In general there are very few use cases where `role=”presentation”` is particularly helpful for assistive technology users, use this role sparingly and thoughtfully.
+
+<figure markdown>
+| Role         | Desktop | Mobile |
+| ------------ | ------- | ------ |
+| button       | 25.20%  | 24.51% |
+| navigation   | 22.97%  | 21.76% |
+| dialog       | 19.01%  | 18.22% |
+| search       | 17.94%  | 17.59% |
+| presentation | 17.83%  | 16.31% |
+
+  <figcaption>
+    {{ figure_link(
+      caption="Top 5 ARIA roles on the web.",
+      sheets_gid="TODO",
+      sql_file="TODO.sql"
+    ) }}
+  </figcaption>
+</figure>
+ 
+#### ARIA Attributes
+
+ARIA attributes can be assigned to HTML elements to enhance the accessibility of the interface. Respecting the first rule of ARIA, they should not be used to achieve something that can be done with native HTML.
+
+##### Labeling and describing elements with ARIA
+
+The browser’s accessibility tree has a computation system that assigns the accessible name (if there is one) to a control, widget, group, or landmark such that it can be announced by assistive technology. There is a specificity ranking that happens to determine which value is assigned to the accessible name. 
+
+The accessible name can be derived from an element’s content (such as button text), an attribute (such as an image `alt` text value), or an associated element (such as a programmatically associated label for a form control. For more information about accessible names see Léonie Watson’s article, [What is an accessible name?](https://developer.paciellogroup.com/blog/2017/04/what-is-an-accessible-name/)
+
```

Comment:
"We can also use ARIA to provide accessible names for elements. There are 2 ARIA attributes that accomplish this, [aria-label](https://www.w3.org/WAI/GL/wiki/Using_aria-label_to_provide_labels_for_objects), [aria-labelledby](https://www.w3.org/WAI/GL/wiki/Using_aria-labelledby_to_provide_a_name_for_user_interface_controls) ."

Those two links are to working documents that are old and pre-date the final spec. Suggest these two documents, which are referenced from WCAG as valid Techniques:
* https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA14.html
* https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA16.html